.github/ISSUE_TEMPLATE/stale_issue_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Stale issue report
+about: Report issues with using the stale action
+title: ''
+labels: bug
+assignees: ''
+---
+
+<!-- Have you tried the [debugging](https://github.com/actions/stale#debugging) section of the readme? -->
+
+## Describe your issue
+
+## Your stale action configuration
+
```

Comment:
I can't make a suggestion due to code parsing issue but after the "## Your stale action configuration" you can provide this predefined template so that it's easier to understand what we need.

"
## Your stale action configuration

```yaml
jobs:
  stale:
    runs-on: ...
    steps:
      - uses: actions/stale@...
        with:
          ...
```
"